### ISSUE TITLE
Limine Wallpaper and Configuration

### ISSUE THREAD

**Issue by pogrammerx** [CosmosOS/Cosmos#2624]
#### Area of Cosmos - What area of Cosmos are we dealing with?
Limine, Cosmos / General

#### Expected Behaviour - What do you think that should happen?
The Limine Wallpaper should be a picture of a flower, but its the default Cosmos one.
I cannot change the limine.cfg file aswell, as it will just be replaced.

#### Actual Behaviour - What unexpectedly happens?
limine.cfg and liminewp.bmp are getting replaced after i change them.

#### Reproduction - How did you get this error to appear?
---

#### Version - Were you using the User Kit or Dev Kit? And what User Kit version or Dev Kit commit (Cosmos, IL2CPU, X#)?
User Kit

**Comment by Kiirx** [comment]
Yes as they are being generated, if you want to change them you have to make a fork sadly

**Comment by 9xbt** [comment]
How could he have limine with the user kit? Latest user kit uses GRUB

**Comment by valentinbreiz** [comment]
Cosmos Gen3 has replaced the IL2CPU-based Gen2 codebase, and `gen3` is now the default branch of this repository. This issue was filed against Gen2, which is no longer developed, so it is being closed as part of the Gen3 cutover (#3088).

If you still hit this on Gen3, please open a fresh issue. The compiler, the build pipeline and the debugging story are all different now, so a new report will get a far more useful answer than reviving this one. Start here: https://cosmosos.github.io/

Gen2 remains on the `gen2` branch, and this issue stays readable.
